store/rootmulti/store_test.go

```diff
@@ -606,6 +606,30 @@ func TestMultiStore_PruningRestart(t *testing.T) {
 	}
 }
 
+func TestUnevenStoresHeightCheck(t *testing.T) {
```

Comment:
can you add a godoc on this test

Reply:
you wants to add more comments on this test

Reply:
I'm very close to finish this, I think I can handle the rest

Reply:
I think he means just add a godoc for the test itself

e.g.
```
// TestUnevenStoresHeightCheck tests for ...
func TestUnevenStoresHeightCheck(t *testing.T) {
```